_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -215,7 +214,17 @@ class HealthCareApp extends React.Component {
   }
 
   handleContinue() {
-    hashHistory.push(this.getNextUrl());
+    const sectionPath = this.props.location.pathname.split('/').filter((path) => { return !!path; });
+    const sectionData = validations.initializeNullValues(_.get(this.state.applicationData, sectionPath));
+    this.publishStateChange(sectionPath, sectionData);
+
+    this.setState({}, () => {
+      if (validations.isValidSection(this.props.location.pathname, sectionData)) {
+        hashHistory.push(this.getNextUrl());
+      } else {
+        document.getElementsByClassName('usa-input-error')[0].scrollIntoView();
```

Comment:
What's happening is `setState()` is not synchronous. See notes https://facebook.github.io/react/docs/component-api.html.  Multiple calls to setState() may get combined into one update to props.  Thus, the error class may not be set when this is called.

I don't know how to fix it "right" but to at least avoid the exception, check that `document.getElementsByClassName('usa-input-error')` isn't empty before trying to get element 0 and do nothing if it is empty. That will yield a bug where the window doesn't scroll but at least we won't throw an exception out of `setState()` which has undefined results.